jax/_src/interpreters/mlir.py

```diff
@@ -254,6 +254,8 @@ def _numpy_array_constant(x: np.ndarray, canonicalize_types
       x = np.array(0 if x.item() == 0 else 0xff, np.uint8)
   elif x.dtype == dtypes.bfloat16:
     x = x.view(np.uint16)
+  elif x.dtype in dtypes._custom_float_dtypes:
```

Comment:
Can we (a) not use an `_`-prefixed name from another module and (b) perhaps assert the size of the element is 1 byte? I'm just worried what happens if someone adds another 2-bytes type to this list or something.

Reply:
Good call - I listed the specific types instead.